Description: do not attempt rendering PDF with Qt image plugin
 Qt 5.15 added Qt PDF module which comes with imageformat plugin, which
 allows rendering first page of PDF document as image.
Origin: upstream, https://github.com/qtwebkit/qtwebkit/commit/0443d42b04e0f99c
Last-Update: 2020-09-13

--- a/Source/WebCore/platform/MIMETypeRegistry.cpp
+++ b/Source/WebCore/platform/MIMETypeRegistry.cpp
@@ -254,6 +254,9 @@ static void initializeSupportedImageMIME
     // Do not treat SVG as images directly because WebKit can handle them.
     supportedImageMIMETypes->remove("image/svg+xml");
     supportedImageResourceMIMETypes->remove("image/svg+xml");
+    // Do not treat PDF as images
+    supportedImageMIMETypes->remove("application/pdf");
+    supportedImageResourceMIMETypes->remove("application/pdf");
 #endif // PLATFORM(QT)
 #endif // USE(CG)
 }
